### DISCUSSION TITLE
Focus on your features, not boilerplates

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1427] [General]
Focus on your features, not boilerplates .Hope more boilerplates ready to use

**Comment by falkoschindler** [comment]
Please explain.

**Reply by v479038280** [reply]
Just one sentence copied from Quasar. The original one is:
>Focus on your features, not the boilerplating
The top-class project initialization with Quasar CLI makes getting started very easy for you as a developer. You can turn your idea into reality in record time. In other words, Quasar does the heavy lifting for you, so you are free to focus on your features and not on boilerplate.

**Reply by falkoschindler** [reply]
Yes, but what is the purpose of your post? Are you suggesting NiceGUI isn't focussing on features? Does the user need to focus too much on boilerplate code? Please be more specific when posting here. To cite from our [FAQ](https://github.com/zauberzeug/nicegui/wiki/FAQs#where-can-i-get-help): "Quite a number of people will read and think about your message. Make it worth their time."
